embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/telemetry/EmbraceTelemetryService.kt

```diff
@@ -21,8 +23,13 @@ internal class EmbraceTelemetryService(
         }
     }
 
+    override fun logStorageTelemetry(fileToSizeMap: Map<String, Int>) {
```

Comment:
I don't think we to store all the file names and the associated size - just the size ALL the files are taking on disk. Instead of passing back a map, we can just add the bytes all up and return a number, which will be a lot easier to consume and make dashboards from.

If we want to get fancy in the the future, we can add different categories (e.g. sessions), but I think one number representing the total amount of cached data we have would be sufficient.